### ISSUE TITLE
Continuous Training Performance Drop

### ISSUE THREAD

**Issue by YiSyuanChen** [nlpyang/PreSumm#154]
Hi all,
Firstly, I would like to thank the authors for giving such excellent works. I have some questions about the training that starts from checkpoint. Below is my validation xent curve, and different color indicates the boundary for continuous training.

![截圖 2020-04-15 下午9 28 33](https://user-images.githubusercontent.com/21264909/79343011-71241700-7f60-11ea-972c-e63ec6de97f8.png)

The setting is identical in the training process, but the performance drops suddenly at the beginning of continuous training. I also check the training xent curve and the learning rate curve to make sure the optimizer is also loaded correctly:

![截圖 2020-04-15 下午9 28 24](https://user-images.githubusercontent.com/21264909/79343558-1b03a380-7f61-11ea-8f17-88c424cbe8d1.png)
![截圖 2020-04-15 下午9 28 40](https://user-images.githubusercontent.com/21264909/79343588-21921b00-7f61-11ea-8839-7b4fd51d10b4.png)

As you may see, the training seems to be normal.  Thus, I would like to search for the help. Does anyone suffer this issue before? Any comment would be very appreciated.
